src/refiners/foundationals/latent_diffusion/schedulers/scheduler.py

```diff
@@ -68,9 +70,13 @@ def _generate_timesteps(self) -> Tensor:
         ...
 
     @property
-    def steps(self) -> list[int]:
+    def all_steps(self) -> list[int]:
         return list(range(self.num_inference_steps))
 
+    @property
+    def diffusion_steps(self) -> list[int]:
```

Comment:
This should be called "denoising_steps" since it corresponds to the reverse diffusion process.

Reply:
Renamed it to "inference_steps" for coherence with "num_inference_steps".